oauth2_http/javatests/com/google/auth/oauth2/ServiceAccountCredentialsTest.java

```diff
@@ -517,6 +583,60 @@ public void refreshAccessToken_failsNotFoundError() throws IOException {
     }
   }
 
+  @Test
+  public void idTokenWithAudience_correct() throws IOException {
+    final String accessToken1 = "1/MkSJoj1xsli0AccessToken_NKPY2";
```

Comment:
google style is to not uise final on local variables unless it's required

Reply:
yah, its all over the place in the existing test case;  i only removed it in the section i added.